src/app.ts

```diff
@@ -63,6 +65,30 @@ const authenticatedRouter = new Router()
         ctx.state.toMember = midQuery.docs[0];
         return await next();
     })
+    .post('/api/members/:mid/request_invite', async ctx => {
```

Comment:
Mid will be changing a bit more often than we anticipated as people sign up incorrectly etc, not just for big life events like marriage. For all centralized google cloud operations, let's stick with uid as the standard for simplicity (and because for some firebase operations makes security more easy).

Reply:
agreed